File: examples/openapi-sdk-petstore/kotlin/sdk/com/example/petstore/models/Pet.kt
package com.example.petstore.models

import kotlinx.serialization.Serializable

@Serializable
public data class Pet(
    public val id: Long? = null,
    public val name: String,
    public val category: Category? = null,
    public val photoUrls: List<String>,
    public val tags: List<Tag>? = null,
    public val status: Pet_Status? = null,
)

File: examples/openapi-sdk-petstore/swift/sdk/Models/Pet.swift
import Foundation

public struct Pet: Codable {
    public let id: Int64?
    public let name: String
    public let category: Category?
    public let photoUrls: [String]
    public let tags: [Tag]?
    public let status: Pet_Status?

    public init(
        id: Int64? = nil,
        name: String,
        category: Category? = nil,
        photoUrls: [String],
        tags: [Tag]? = nil,
        status: Pet_Status? = nil
    ) {
        self.id = id
        self.name = name
        self.category = category
        self.photoUrls = photoUrls
        self.tags = tags
        self.status = status
    }
}
